services/basic_types.proto

```diff
@@ -113,6 +113,21 @@ message AccountID {
     }
 }
 
+/**
+ * Represents an NFT on the Ledger
+ */
```

Comment:
Docs can be taken from the removed `UniqueTokenId` . They seem to be detailed

Reply:
Fixed